### ISSUE TITLE
Uploader for classification label only

### ISSUE THREAD

**Issue by jnothman** [Sydney-Informatics-Hub/Weed-AI#490]
Provide an uploader UI where the user uploads a spreadsheet (CSV with header, to be precise) with at least columns "filename" and "category". Then: Category mapper, agcontext, metadata, images.

**Comment by jnothman** [comment]
Note this would need to handle the case where the same filename appears on multiple rows with different category

**Comment by jnothman** [comment]
This is something Michael wants. Do you want to write the weedcoco.importers script, or should I?

**Comment by ElevnLi** [comment]
> This is something Michael wants. Do you want to write the weedcoco.importers script, or should I, @ElevnLi ?

I don't quite understand this issue.. This csv file will be converted to coco?

**Comment by jnothman** [comment]
Yeah, something like https://github.com/AlexOlsen/DeepWeeds/blob/master/labels/labels.csv will be converted to COCO.

**Comment by jnothman** [comment]
I've not thought about how to handle the negative clas..
